rotkehlchen/db/drivers/server.py

```diff
@@ -117,15 +104,22 @@ def __init__(self, port: int) -> None:
         # to avoid sending back error from the server while terminating.
         self.shutting_down = False
 
-    def _handle_connection(self, call_data: ZMQCallData) -> None:
-        connect_func = sqlite3.connect if call_data.connection_type.value == DBConnectionType.GLOBAL.value else sqlcipher.connect  # noqa: E501  # pylint: disable=no-member
-        self.db_connections[call_data.db_path] = connect_func(*call_data.args, **call_data.kwargs)
+    def _handle_connection(
+            self,
+            call_data: db_messages_pb2.ZMQCallData,  # type: ignore[name-defined]  # pylint: disable=no-member
+            args: tuple[Any],
+            kwargs: dict[str, Any],
+    ) -> None:
+        connect_func = sqlite3.connect if call_data.connection_type == db_messages_pb2.DBConnectionType.GLOBAL else sqlcipher.connect  # type: ignore[attr-defined]  # pylint: disable=no-member  # noqa: E501
+        self.db_connections[call_data.db_path] = connect_func(*args, **kwargs)
 
     def _handle_query(
             self,
             db_connection: UnderlyingConnection,
-            call_data: ZMQCallData,
-    ) -> ZMQReturnData:
+            call_data: db_messages_pb2.ZMQCallData,  # type: ignore[name-defined]  # pylint: disable=no-member
+            args: tuple[Any],
```

Comment:
I would use the type of what is allowed for args and kwargs... not anything is valid